src/components/auth/AuthForm.jsx

```
@@ -1,10 +1,13 @@
 import { useEffect, useState } from 'react';
 import { useForm } from 'react-hook-form';
-import { Button, Input, Link } from "@heroui/react";
+import { Button, Input, Link } from '@heroui/react';
 import { Link as RouterLink } from 'react-router-dom';
 import { PiWarningBold } from 'react-icons/pi';
 import { useLoginUser, useRegisterUser } from '../../hooks/react-query/user/useUser';
 import Logo from '../Logo';
+import { supabaseClient } from '../../lib/supabase.js';
```

ESLint

[error] 'supabaseClient' is defined but never used.

(no-unused-vars)